docs/Migrating.md

```diff
@@ -9,6 +9,12 @@
   packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used
   to interact directly with a Unity environment, while `mlagents.trainers`
   contains the classes for training agents.
+* The supported Unity version has changed from `2017.1 or later` to `2017.4.10f1 or later`. The lower versions might still work, but you may encounter error that looks like this:
```

Comment:
Do we want 2017.4 here as well?

Reply:
Cap line length to 80 characters (minus code samples and things that need to be verbatim).